apps/sms/test/marionette/composer_test.js

```diff
@@ -258,4 +258,42 @@ marionette('Messages Composer', function() {
       assertIsFocused(composer.messageInput, 'Message input should be focused');
     });
   });
+
+
+  var Messages = require('./lib/messages');
```

Comment:
Looks like ```Messages``` already loaded above; Can we move ```InboxView``` require there as well so that all dependencies are clearly visible?